### PR TITLE
Fix integration tests failing due to missing Jupyter plugin

### PR DESCRIPTION
**Pull request by rli**
## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test-scripts/integrationTest.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ async function setupVSCode(): Promise<string> {
         process.env.AWS_TOOLKIT_IGNORE_WEBPACK_BUNDLE = 'true'
 
         const disableExtensions = await getCliArgsToDisableExtensions(vsCodeExecutablePath, {
```
Comment (rli) [inline test-scripts/integrationTest.ts:33]: should we even do this? seems pointless...

Comment (bryceitoc9) [reply test-scripts/integrationTest.ts:33]: I think this is mainly for running locally instead of via CI.

**Review by bryceitoc9 — APPROVED** [review]

File: src/shared/extensions.ts [hunk 2]
```diff
@@ -14,6 +14,8 @@ import { CredentialsStore } from '../credentials/credentialsStore'
 export const VSCODE_EXTENSION_ID = {
     awstoolkit: 'amazonwebservices.aws-toolkit-vscode',
     python: 'ms-python.python',
+    // python depends on jupyter plugin
+    jupyter: 'ms-toolsai.jupyter',
```
Comment (justinmk3) [inline src/shared/extensions.ts:18]: just curious (also maybe good to add to the comment), but what part of python functionality depends on jupyter? was not needed for existing integration tests...

File: test-scripts/integrationTest.ts (hunk 1 above)
Comment (justinmk3) [reply test-scripts/integrationTest.ts:33]: as Bryce said, but also to be explicit about what is actually needed. Otherwise we have hidden dependencies

**Comment by justinmk3** [comment]
Seeing `Unknown dependency 'ms-toolsai.jupyter'` error in https://github.com/aws/aws-toolkit-vscode/pull/1393 . I wonder if it's a race condition somewhere ... 

```
globalSetup: after()
  28 passing (4s)
  1 failing
  1) SAM Integration Tests
       "before all" hook in "SAM Integration Tests":
     Error: Unknown dependency 'ms-toolsai.jupyter'
      at h._handleActivateRequest (.vscode-test/vscode-1.51.1/VSCode-linux-x64/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:665:608)
      at h._activateExtensions (.vscode-test/vscode-1.51.1/VSCode-linux-x64/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:666:71)
      at h.activateById (.vscode-test/vscode-1.51.1/VSCode-linux-x64/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:664:70)
      at p._activateById (.vscode-test/vscode-1.51.1/VSCode-linux-x64/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:929:571)
      at p.activateByIdWithErrors (.vscode-test/vscode-1.51.1/VSCode-linux-x64/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:929:629)
      at ye.activate (.vscode-test/vscode-1.51.1/VSCode-linux-x64/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1016:217)
      at Object.<anonymous> (src/integrationTest/integrationTestsUtilities.ts:22:25)
      at Generator.next (<anonymous>)
      at /codebuild/output/src713431629/src/github.com/aws/aws-toolkit-vscode/dist/src/integrationTest/integrationTestsUtilities.js:12:71
      at new Promise (<anonymous>)
      at __awaiter (dist/src/integrationTest/integrationTestsUtilities.js:8:12)
      at Object.activateExtension (dist/src/integrationTest/integrationTestsUtilities.js:22:12)
      at /codebuild/output/src713431629/src/github.com/aws/aws-toolkit-vscode/src/integrationTest/sam.test.ts:102:11
```

**Comment by justinmk3** [comment]
Oops, I see now that this PR actually solves the CI failures in https://github.com/aws/aws-toolkit-vscode/pull/1393 , not causes them.

**Pull request merged**